LDM-148.tex

```diff
@@ -171,23 +171,22 @@ \section{Component Overview}\label{component-overview}
 components. The top tier is the LSST Science Platform, which is deployed
 in the DACs and other computational environments to provide a user
 interface and analysis environment for science users and LSST staff. The
-detailed design of this tier is given in ``LSST Science Platform,'' \citeds{LDM-542}. The next
+detailed design of this tier is given in ``LSST Science Platform'' (\citeds{LDM-542}). The next
```

Comment:
The other option here, and in other places in the document, is to use `\citedsp{}`